back-office/design/image-settings/regenerate-thumbnails.md

```diff
@@ -1,3 +1,13 @@
+# SPECIFICATIONS - DESING > IMAGE SETTINGS
+
+## IMAGE SETTINGS
+
+## IMAGE GENERATION OPTIONS
+
+## REGENERATE THUMBNAILS
```

Comment:
Added in different files, cf. https://github.com/PrestaShop/prestashop-specs/pull/179. Note that, for the image generation options, checkboxes appear in 1.7.7.x when in a single shop context or in a group context.